### DISCUSSION TITLE
Ipban pro datacenter

### DISCUSSION THREAD

**Discussion by H3jake** [DigitalRuby/IPBan#258] [Q&A] [answered]
is there an easy way to deploy this, i know there is a powershell script for for ipban but cant see one for ipbanpro?

**Comment by jjxtra** [comment, marked as answer]
You can run with argument of -help for automated installing documentation. The most common parameters are: -install -v=serverurl -user=username -password=password

**Comment by H3jake** [comment]
Awesome, thank you!

Best Regards,
Jake Hardy
Technical Consultant
Habitat3 | i360 Cloud

Habitat3 | 1300 550 941 | i360 Cloud | 1800 38 25 80 |
Websites:  www.habitat3.net.au<http://www.habitat3.net.au/> | www.i360cloud.net.au<http://www.i360cloud.net.au/>
Office: Level 5, 20 Bond Street, Sydney NSW 2000
_________________________________________________
